move orioledb to extensions schema

samrose opened the pull request:
## What kind of change does this PR introduce?

Moves the orioledb extension to the extensions schema if:

orioledb is enabled in the public schema
the extensions schema exists
This is important because orioledb exposes some entities that we don't necessarily want exposed over APIs by default

Recommending we use this PR instead of https://github.com/supabase/postgres/pull/1433 as it is more up to date CI, less hassle toward reconciling git changes

samrose committed:
chore: move orioledb to extensions scehama

samrose committed:
ore: update schemas

samrose committed:
chore: create testing AMI images

samrose committed:
Merge branch 'develop' into sam/oriole-extensions-schema

darora reviewed (approved):
Does this work as expected with restores and unpauses?

samrose commented:
@darora thanks for the approval, I should have noted that we are going to hold off on merging this until I test restore, unpause. That should be done by end of day tomorrow, and then will set an actual vars.yml release version, and pr for rollout. Oliver stated that this does not require rollout to existing instances. But our policy now is to always test pause/unpause and restore, so will be doing that for sure.

samrose committed:
chore: bump version for release

This change targets orioledb
moving installed orioledb extension from "public" schema to "extensions" schema

samrose commented:
tested and confirmed no failures on pause/restore. Per @olirice this doesn't actually need to apply the sql migration on an existing project. But the migration succeeds on new project.
